nifi-framework-bundle/nifi-framework/nifi-framework-components/src/main/java/org/apache/nifi/controller/service/StandardControllerServiceNode.java

```diff
@@ -759,7 +759,7 @@ private void invokeDisable(ConfigurationContext configContext) {
             final Throwable cause = e instanceof InvocationTargetException ? e.getCause() : e;
             final ComponentLog componentLog = new SimpleProcessLogger(getIdentifier(), StandardControllerServiceNode.this, new StandardLoggingContext(StandardControllerServiceNode.this));
             componentLog.error("Failed to invoke @OnDisabled method due to {}", cause);
```

Comment:
This line should also be changed:
```suggestion
            componentLog.error("Failed to invoke @OnDisabled method", cause);
```